wolfssl/wolfcrypt/port/maxim/maxq10xx.h

```diff
@@ -0,0 +1,142 @@
+/* maxq10xx.h
```

Comment:
I don't see where this new file is handled in an include.am.

Reply:
Ah!! Thank you! This might solve a problem I've been seeing!!

Reply:
Fixed by https://github.com/anhu/wolfssl/pull/9/commits/e74d258c87e7db9d5c1827c71dad8530751137de .